src/main/java/frc/robot/Robot.java

```diff
@@ -1,17 +1,45 @@
 package frc.robot;
 
 import com.chopshop166.chopshoplib.commands.CommandRobot;
+import com.chopshop166.chopshoplib.controls.ButtonXboxController;
+
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.maps.RobotMap;
+import frc.robot.subsystems.Climber;
 
 public class Robot extends CommandRobot {
 
+  private Command autonomousCommand;
```

Comment:
This and the chooser are not necessary - for `CommandRobot`, the only thing you need to do in order to make an autonomous is:

```java
@Autonomous
public final CommandBase auto1 = foobar();
```